yuhow/yuhowNote.tex

```diff
@@ -33,27 +82,156 @@
+\section{Formula derivation}
+ \label{sec:formula_derivation}
+Noting some important deriviations of equations listed in Prof. S. C. Chang's CESE paper~\cite{CESE_Shin_Chung_Chang_1995}
  \begin{enumerate}
-  \item Why does Eq. (2.4) imply Eq. (2.5). ?
-  \item Why can Eq. (2.9) be proved using the fact that the total flux of $h^{*}$ leaving the boundary of any space-time region that is the union of any combination of CEs vanishes? Can't figure it out.
-  \item What's the finite-difference appromixation?
-  \item What's the meaning of "the $\alpha-\mu$ scheme uses a mesh that is staggered in time"?
-  \item What's the Lax scheme?
-  \item What's the amplification factors? Also what's their meaning/usage in the Leapfrog/DuFort-Frankel scheme?
-  \item What's the meaning of "two-level" and "three-level" scheme?
-  \item Why does not solutions of Eq. (2.22) dissipate with time? Or why is "no dissipation" equivalent to "neutrally stable"?
-  \item Why the total flux leaving any conservation element is zero? This question is relevant to the definition of Eq. (2.28).
-  \item Why does the term "$-\mu\partial^{2}u^{*}(x, t; j, n)/\partial x^{2} $ vanish "in Eq. (2.29)?
-  \item (In page 303) Why is the condition that Eq. (2.30) being valid uniformly within an SE stronger than Eq. (2.28) for a higher-order expansion? Are they the same thing but just in different form?
-  \item Eq. (2.33) maybe is wrong. The partial derivative in the left-hand side of Eq. (2.33) should be with respect to x instead of t. Will try to double check with author though email. (STATUS: waiting for Prof. Chang's reply)
+  \item Derivation of Eq.(2.5) $(u_t)^n_j = -a(u_x)^n_j$:
+
+        \hspace{4mm}Substituting Eq.(2.3) into Eq.(2.1), then you can get Eq.(2.5).
+
+  \item Derivation of Eq.(2.12) $\frac{4}{(\Delta x)^{2}}F_{\pm}(j, n)=\pm(\frac{1}{2})[(1-\nu^{2}+\xi)(u_{x})^{n}_{j}+(1-\nu^{2}-\xi)(u_{x})^{n-1/2}_{j\pm1/2}]+\frac{2(1\mp\nu)}{\Delta x}(u^{n}_{j}-u^{n-1/2}_{j\pm1/2})$, where $\nu\equiv\frac{a\Delta t}{\Delta x}$ and $\xi\equiv\frac{4\mu\Delta t}{(\Delta x)^{2}}$:
+        \begin{figure}[hbtp]
+          \centering
+            \includegraphics[width=0.8\textwidth]{mesh_and_ce_se_v1.pdf}
+            \caption{The mesh, conservation elements and solution elements. The cross-shaped region formed with red dash line is one of the solution element in the mesh.}
+          \label{fig:mesh_and_ce_se_v1}
+        \end{figure}
+
+        \hspace{4mm}Eq.(2.6) $u^{*}(x,t;j,n)=u^{n}_{j}+(u_{x})^{n}_{j}[(x-x_{j})-a(t-t^{n})], (x,t)\in SE(j,n)$
+
+        \hspace{4mm}Eq.(2.9) $\vec{\bf{g}}^{*}\equiv(-u^{*},au^{*}-\mu\partial u^{*}/\partial x), d{\bf{r}}\equiv(dx,dt)$
+
+        \hspace{4mm}$F_{\pm}(j,n)=\oint^{c.c.}_{S(CE_{\pm}(j,n))}\vec{\bf{g}}^{*}\cdot d{\bf{r}}$
+
+        \hspace{4mm}For CE$_{-}(ABPC)$ shown in Figure~\ref{fig:mesh_and_ce_se_v1}:
+
+        \hspace{8mm}$\int^{B}_{A}\{-u^{n}_{j}-(u_{x})^{n}_{j}[(x-x_{j})-a(t-t^{n})]\}dx$
```

Comment:
I think display format is more readable than the inline format.  Although we start from scratch, the end results hopefully can be useful for later readers.  The inline math is pretty hard to maintain in LaTeX.